e2e/cypress/integration/enterprise/system_console/channel_moderation_spec.js

```diff
@@ -35,24 +59,876 @@ const enableAllChannelModeratedPermissions = () => {
+            // # Create a new team override scheme and remove manage members option for members
+            goToPermissionsAndCreateTeamOverrideScheme('manage_members');
+            deleteOrEditTeamScheme('manage_members', 'edit');
+            cy.get('#all_users-public_channel-manage_public_channel_members').click();
+            saveConfig(false);
+            cy.wait(1000); //eslint-disable-line cypress/no-unnecessary-waiting
+
+            // * Assert that Manage Members is disabled for members and a message is displayed
+            visitAutemChannelConfigPage();
+            cy.findByTestId('admin-channel_settings-channel_moderation-manageMembers-disabledMember').
+                should('exist').
+                and('have.text', 'Manage members for members are disabled in manage_members Team Scheme.');
+            cy.findByTestId(checkboxesTitleToIdMap.MANAGE_MEMBERS_MEMBERS).should('be.disabled');
+            cy.findByTestId(checkboxesTitleToIdMap.MANAGE_MEMBERS_GUESTS).should('not.exist');
+
+            visitAutemChannel('user-1');
+            viewManageChannelMembersModal('View');
+
+            // * Add Members button does not exist in manage channel members modal
+            cy.get('#showInviteModal').should('not.exist');
+        });
+    });
+
+    describe('MM-23102 - Channel Mentions', () => {
+        it('Channel Mentions option for Guests', () => {
+            // # Uncheck the Channel Mentions option for Guests and save
+            visitAutemChannelConfigPage();
+            disableChannelModeratedPermission(checkboxesTitleToIdMap.CHANNEL_MENTIONS_GUESTS);
+            saveConfig();
+
+            visitAutemChannel('guest');
+
+            // # Check Guest user has the permission to user special mentions like @all @channel and @here
+            postChannelMentionsAndVerifySystemMessageExist();
+
+            // # Visit Channel page and Search for the channel.
+            visitAutemChannelConfigPage();
+
+            // # check the channel mentions option for guests and save
+            enableChannelModeratedPermission(checkboxesTitleToIdMap.CHANNEL_MENTIONS_GUESTS);
+            saveConfig();
+
+            visitAutemChannel('guest');
+
+            // # Check Guest user has the permission to user special mentions like @all @channel and @here
+            postChannelMentionsAndVerifySystemMessageNotExist();
+        });
+
+        it('Channel Mentions option for Members', () => {
+            // # Visit Channel page and Search for the channel.
+            visitAutemChannelConfigPage();
+
+            // # Uncheck the channel mentions option for guests and save
+            disableChannelModeratedPermission(checkboxesTitleToIdMap.CHANNEL_MENTIONS_MEMBERS);
+            saveConfig();
+
+            visitAutemChannel('user-1');
+
+            // # Check Member user has the permission to user special mentions like @all @channel and @here
```

Comment:
TYPO: Comment should say `Check Member user does not has the permission to use special mentions...`

Reply:
Good catch. Thanks 👍